src/global.c

```diff
@@ -0,0 +1,4 @@
+#include "global.h"
```

Comment:
Uf, no I should remove the global.c. I made a mistake first, I guess I accidentally pushed it to git.